tests/h/search/query_test.py

```diff
@@ -714,15 +716,18 @@ def test_visibility_of_moderated_and_nipsaed_annotations(
     ):
         pyramid_request.user = user
         search.append_modifier(query.HiddenFilter(pyramid_request))
+
         presenter = AnnotationSearchIndexPresenter.return_value
         presenter.asdict.return_value = {"id": "ann1", "hidden": hidden, "nipsa": nipsa}
         Annotation(id="ann1")
         presenter.asdict.return_value = {"id": "ann2", "hidden": False, "nipsa": False}
         Annotation(id="ann2", userid=user.userid)
+
+        result = search.run({})
+
```

Comment:
I swapped this about as before it was doing a bunch of expectation stuff before getting the result. This is hopefully a bit more given/when/then.